SDK_dotnet/Models/PayLoad.cs

```diff
@@ -1,7 +1,44 @@
-﻿namespace Gluwa.Models
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Gluwa.Models
```

Comment:
I don't think you need this class actually. The only webhook we send at this moment is TransactionConfirmed, TransactionFailed and TransactionCreated. And for all webhooks, we only use `WebhookPayLoad` you have below